src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -556,7 +531,7 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
                 TimeSpan timerInterval = timer == null ? TimeSpan.MinValue : TimeSpan.FromMilliseconds(timer.Interval);
                 try
                 {
-                    await InvokeAsync(invoker, invokeParameters, timeoutTokenSource, functionCancellationTokenSource,
+                    await InvokeAsync(invoker, parameterHelper, timeoutTokenSource, functionCancellationTokenSource,
```

Comment:
Correct. @hhawkins  may need to add a new method on the helper to expose exactly what he needs, but the key information is all in one place now. 


---
In reply to: [125711434](https://github.com/Azure/azure-webjobs-sdk/pull/1209#discussion_r125711434) [](ancestors = 125711434)